cuegui/cuegui/MenuActions.py

```diff
@@ -466,40 +484,78 @@ def eatDead(self, rpcObjects=None):
             if cuegui.Utils.questionBoxYesNo(self._caller, "Confirm",
                                              "Eat all DEAD frames in selected jobs?",
                                              [job.data.name for job in jobs]):
+                blocked_job_owners = []
                 for job in jobs:
-                    job.eatFrames(state=[opencue.compiled_proto.job_pb2.DEAD])
+                    # check permissions
+                    if not cuegui.Utils.isPermissible(job):
+                        blocked_job_owners.append(job.username())
+                    else:
+                        job.eatFrames(state=[opencue.compiled_proto.job_pb2.DEAD])
+                if blocked_job_owners:
+                    cuegui.Utils.showErrorMessageBox(
+                        AbstractActions.USER_INTERACTION_PERMISSIONS.format(
+                            "eat dead for some of the selected jobs",
```

Comment:
eat dead **frames**:

`"eat dead for some of the selected jobs"` -> `"eat dead frames for some of the selected jobs"`